src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int length;
+    private int width;
+
+    public Rectangle(String color, int length, int width) {
+        super(color);
+        this.length = length;
+        this.width = width;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.round(length * width * 1.0);

```

Comment:
1. remove round
2. why did you multiply by 1.0?

```suggestion4
        return length * width;
```